### PR TITLE
Chore/new tokens docs

### PR DESCRIPTION
**Pull request by NoSQLKnowHow**
The code that is embedded in this page is there temporarily and will be brought into line with the rest of the docs at a later time once the other PRs are merged to the SDK.

### CODE REVIEW

File: static/img/momento-auth-tokens.png
Comment (krispraws) [inline static/img/momento-auth-tokens.png:1]: Should `Tokens` on second line be a singular `Token`?

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md]: It can also be an object that satisfies the CacheName interface ` { name: string }`. The string is a shorthand. We may add other types of selectors like Prefix in future.
Similar comment for `topic` which can be a TopicName

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md]: typo - should be `AllCaches`

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md]: See my other comment about `CacheName` / `TopicName` interfaces

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md]: missing `are` in `then assigned`?

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md]: I think I used `Array` in my draft too but List might be a more generic term.

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md:120]: We shouldn't have code directly embedded here - I think you already pointed out it is temporary but just reiterating so we don't forget.

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md:115]: Should cache be highlighted with backticks in `defined in cache`?

File: docs/develop/api-reference/auth-tokens.md
Comment (mjindalmotiv8) [inline docs/develop/api-reference/auth-tokens.md:162]: Another question worthwhile adding is that we do now allow tokens with write only or publish only permissions

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md]: It can also be a duration defined as minutes or hours by using ExpiresIn.minutes() or ExpiresIn.hours()

File: docs/develop/api-reference/auth-tokens.md
Comment (NoSQLKnowHow) [reply docs/develop/api-reference/auth-tokens.md:162]: I will add this.

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md]: I think `one or more permissions` is more accurate than `one or more roles`

File: docs/develop/api-reference/auth-tokens.md
Comment (mjindalmotiv8) [reply docs/develop/api-reference/auth-tokens.md:162]: Thank you

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md]: Still conflicted about using `Authentication` here vs `Auth` to denote both AuthZ and AuthN

**Review by krispraws — COMMENTED** [review]
Left some comments but looks good overall. There is also a matrix of which SDKs support which features. We need to update that. I'll look into the details of that.

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [reply docs/develop/api-reference/auth-tokens.md:120]: docs code snippets PR: https://github.com/momentohq/client-sdk-javascript/pull/623

File: static/img/momento-auth-tokens.png
Comment (NoSQLKnowHow) [reply static/img/momento-auth-tokens.png:1]: Yes. Good catch! changing now.

File: docs/develop/api-reference/auth-tokens.md
Comment (NoSQLKnowHow) [reply docs/develop/api-reference/auth-tokens.md]: Not saying no, but for simplicity's sake, why would anyone ever use the object when they could just use a simple string? My thought is if in the future we change things, then we could put that in. Thoughts?

File: docs/develop/api-reference/auth-tokens.md
Comment (NoSQLKnowHow) [reply docs/develop/api-reference/auth-tokens.md:120]: Yep. There is a plan to get this using the SDK repo examples, but we need to kick this out the door.

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [reply docs/develop/api-reference/auth-tokens.md]: Because I am not sure we can expose this as a string in all SDKs - we may end up having CacheName/TopicName classes in other languages.

Comment (krispraws) [reply docs/develop/api-reference/auth-tokens.md]: This documentation is common to all SDKs so it should define the fields logically rather than use specific data types.

File: docs/develop/api-reference/auth-tokens.md
Comment (NoSQLKnowHow) [reply docs/develop/api-reference/auth-tokens.md]: Correct. changed.

File: docs/develop/api-reference/auth-tokens.md
Comment (NoSQLKnowHow) [reply docs/develop/api-reference/auth-tokens.md]: added

File: docs/develop/api-reference/auth-tokens.md
Comment (NoSQLKnowHow) [reply docs/develop/api-reference/auth-tokens.md]: Changed to auth.

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md]:
```suggestion
| expiresIn       | Number \| ExpiresIn object | The number of seconds until the token expires or an ExpiresIn object representing a duration by calling the `ExpiresIn.never()`, `ExpiresIn.seconds()`, `ExpiresIn.minutes()`, or `ExpiresIn.hours()` methods. |
```

File: docs/develop/api-reference/auth-tokens.md
Comment (NoSQLKnowHow) [reply docs/develop/api-reference/auth-tokens.md]: That makes far more sense to me and was not clear before. will change!

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md]:
```suggestion
title: Access control token API reference information
```

File: docs/develop/api-reference/auth-tokens.md
Comment (NoSQLKnowHow) [reply docs/develop/api-reference/auth-tokens.md]: Fixed.

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md]:
```suggestion
sidebar_label: Access control tokens
```

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md]:
```suggestion
description: Learn the access control token API calls you need to know about and how to use them with Momento services.
```

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md]:
```suggestion
# Access control token API reference
```

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md]:
```suggestion
These APIs are used to manage Momento access control tokens and access. Tokens can be scoped to have one or more permissions to give access to one or more caches or topics.
```

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md:23]:
```suggestion
Generates a new Momento authentication and authorization token for access control with the specified permissions and expiry, henceforth referred to as auth token.
```

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md]:
```suggestion
These objects define the specific role with cache or topic information and are assigned to a [TokenScope](#tokenscope). 
```

**Review by krispraws — COMMENTED** [review]
Text change suggestions

File: docs/develop/api-reference/auth-tokens.md
Comment (NoSQLKnowHow) [reply docs/develop/api-reference/auth-tokens.md]: I thought you wanted things to say Auth.

File: docs/develop/api-reference/auth-tokens.md
Comment (NoSQLKnowHow) [reply docs/develop/api-reference/auth-tokens.md]: ok, this one is a problem. most everywhere in the docs we call these tokens "Momento auth tokens", not Momento access control tokens. Has this changed and we need to change this everywhere?

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [inline docs/develop/api-reference/auth-tokens.md]: @danielamiao - are you ok with using Access Control Tokens as the main term? cc @rishtigupta and @bruuuuuuuce for the console wording

Comment (danielamiao) [reply docs/develop/api-reference/auth-tokens.md]: Fine Grained Access Control (FGAC) is better

File: docs/develop/api-reference/auth-tokens.md
Comment (krispraws) [reply docs/develop/api-reference/auth-tokens.md]: We cannot get rid of the `auth token` terminology. It is everywhere including api names. I suggest we add a transition phrase like `Momento authentication and authorization token for fine grained access control, henceforth referred to as auth token` at the beginning of the page.

**Review by mjindalmotiv8 — DISMISSED** [review]
Approved from my side !! Once Ramya is ok, approved from engineering

File: docs/develop/api-reference/auth-tokens.md
Comment (NoSQLKnowHow) [reply docs/develop/api-reference/auth-tokens.md]: Discussed offline, resolving.

**Review by krispraws — APPROVED** [review]

**Pull request merged**
